driver-core/src/main/com/mongodb/client/model/Filters.java

```diff
@@ -208,17 +208,15 @@ private Filters() {
     }
 
     /**
-     * Creates a filter that performs a logical AND of the provided list of filters.  Note that this will only generate a "$and"
-     * operator if absolutely necessary, as the query language implicity ands together all the keys.  In other words, a query expression
-     * like:
+     * Creates a filter that performs a logical AND of the provided list of filters.
```

Comment:
Also apply this to the overloaded method just below.

Reply:
Done